generator/flat/templates/lookup.go

```diff
@@ -57,3 +58,15 @@ func Evaluate(templateStr string, obj interface{}) (string, error) {
 
 	return buf.String(), nil
 }
+
+func convertByteToInitForm(b []byte) string {
```

Comment:
A Go string can encode any byte slice just by using the `%q` formatting verb, like `fmt.Sprintf("%q", []byte{0x01, 0xfd})`. Ex. https://play.golang.org/p/V-JgdJce9Eu

Reply:
I did the change to use this. As a bonus we can use the global as Const, because I can define it as string and do casting to []byte at the function level.